lib/services/local/access_list.go

```diff
@@ -303,6 +320,84 @@ func (a *AccessListService) GetAccessListMember(ctx context.Context, accessList
 	return member, trace.Wrap(err)
 }
 
+// GetAccessListOwners returns a list of all owners in an Access List, including those inherited from nested Access Lists.
+//
+// Returned Owners are not validated for ownership requirements – use `IsAccessListOwner` for validation.
+func (a *AccessListService) GetAccessListOwners(ctx context.Context, accessListName string) ([]*accesslist.Owner, error) {
```

Comment:
Does this have any UT coverage ?

Reply:
It should be indirectly tested via some other unit tests but I'll verify.